### ISSUE TITLE
Projects Page: Update alt text for project card images to adhere to WCAG

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3861]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Projects` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the `project.alt` property's value within `/assets/js/current-projects.js`:

**From:**
```
"alt": {{ project.alt }}
```

**To:**
```
"alt": ""
```

- [x] Ensure that the corresponding `Projects` page stays the same after the change
- [x] Using developer tools to inspect the image, ensure that the new alt text gets incorporated after the change.
  - **Note:** `alt=""` in code while show up as `alt` when using developer tools to inspect the image's alt text property.

### Resources/Instructions
- Projects Page: https://www.hackforla.org/projects/

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `/assets/js/current-projects.js`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/910e145af61f4072aa8ebb24da385197729af942/assets/js/current-projects.js#L89

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by ExperimentsInHonesty** [comment]
@codinghorizons, I see that this issue was closed by @mademarc the day after you self-assigned Mar 27, 2023.  And after that you didn't do another good first issue.  I am not sure what happened, but in case you still want to be on the team please go ahead and pick up another [good first issue from the prioritized backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22#column-7198257)

**Comment by w1ld-r** [comment]
Availability: M-F 1pm-8pm CST
ETA: 2023-02-13 by 8pm CST

**Comment by w1ld-r** [comment]
FYI: In my code, the only instance of {{ project.alt }} on current-projects.js was on line 123, while the ticket specified line 89. Since there was only one instance on the page, I went ahead with the changes.
